### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by zietbukuel** [Codeinwp/Nivo-Slider-jQuery#412]
It hasn't been updated in like 4 years now and the last release was 7 years ago...

**Comment by makarovpro** [comment]
Yes, it's very old already.

**Comment by thecarnie** [comment]
There are forks with some fixes and updates applied --

**Comment by TimexPeachtree** [comment]
Yep, buts good news its being used on live active projects like [NOP Commerce](https://www.nopcommerce.com/en)  ;)

**Comment by thecarnie** [comment]
As time permits, I attempt to round up fixes that are submitted here and apply them to my fork.  I am somewhat behind.  I will work on this --

**Comment by thecarnie** [comment]
The beauty is it's VERY STABLE - and other than issues that arise with newer jQuery versions, it continues to run --

I keep two different sliders/carousels maintained due to my usage in personal projects over the years -- they're both lightweight and stable - and in that sense works great for production.
